### ISSUE TITLE
finetune clip both can get the better result

### ISSUE THREAD

**Issue by Code-kunkun** [ABaldrati/CLIP4Cir#5]
Hi, thank you for your amazing work！
I just have a question. I found that finetune clip both at the first stage can attain better result. But why you didn't mention it at your paper?

**Comment by ABaldrati** [comment]
Hi!
Thanks for your interest in our work!!

To answer your question... Yes, fine-tuning both encoders can lead to better results in both datasets.

Unfortunately, when we performed the experiments (for the already published papers) we had access only to a low-memory GPU. This means that when we wanted to fine-tune both encoders we had to lower (a lot) the batch size. 
And, since we use a contrastive loss during training, a small batch size leads to poor performance.

Long story short: a larger batch size fine-tuning only the text encoder brings better performance than a smaller batch size fine-tuning both encoders.

We have recently submitted a paper (we are currently under review) where we perform any kind of performance comparison and ablation study. 

If you need more info do not hesitate to ask!

**Comment by Code-kunkun** [comment]
thank you for your reply!!!
